output/cloud/data.go

```diff
@@ -116,16 +116,16 @@ func NewSampleFromTrail(trail *httpext.Trail) *Sample {
 	}
 
 	values := make(map[string]float64, length)
-	values[metrics.HTTPReqs.Name] = 1
-	values[metrics.HTTPReqDuration.Name] = stats.D(trail.Duration)
-	values[metrics.HTTPReqBlocked.Name] = stats.D(trail.Blocked)
-	values[metrics.HTTPReqConnecting.Name] = stats.D(trail.Connecting)
-	values[metrics.HTTPReqTLSHandshaking.Name] = stats.D(trail.TLSHandshaking)
-	values[metrics.HTTPReqSending.Name] = stats.D(trail.Sending)
-	values[metrics.HTTPReqWaiting.Name] = stats.D(trail.Waiting)
-	values[metrics.HTTPReqReceiving.Name] = stats.D(trail.Receiving)
+	values[metrics.HTTPReqsName] = 1
```

Comment:
You can have access to the built-in metrics here (through `WithBuiltinMetrics`), right? So why not use the `Name` properties from them and avoid the constants?

Reply:
Because unfortunately in some places I don't have a context in this tests, so after I tried for an hour to add it everywhere and failed I decided there is no point int making it harder for myself. There are way too many places where using builtingMetrics in the tests is way too hard for the (IMO) no benefit of not having the names as consts that I prefer to skip adding a bunch of lines. Like right here this is a function it doesn't have access to anything and it's tests don't have access to anything and before I stopped adding stuff I was at 100 lines+ of changes just to  change that and have working tests.

Again I am pretty sure a lot of this code will be changes *again* and *again* while we do the remaining metric changes, so making it "better" in practically any way that adds more code or is harder seems like wasted time for me

Reply:
Hmm, yeah, I can see how it will be tricky in some of the standalone tests. That said, I'd probably prefer if we use naked strings in these tests instead, since we're only only verifying that some metric exists or something like that... :man_shrugging: 

Having the names as exported consts or not doesn't functionally matter, but the current way is more misleading than helpful IMO. It makes it seem OK to use the constants in normal non-test code, like here, when an explicit dependency through `WithBuiltinMetrics` will be easy and much cleaner. Still, as I mentioned, this is a relatively minor nitpick, and we can easily fix it later if we decide to, so I'm not very insistent on changing it now.